src/client/sandbox/code-instrumentation/location/index.js

```diff
@@ -34,11 +37,17 @@ export default class LocationAccessorsInstrumentation extends SandboxBase {
 
         window[INSTRUCTION.setLocation] = (location, value) => {
             if (isLocation(location)) {
-                location = value;
+                var resourceType = null;
 
-                return location;
-            }
+                if (window.self !== window.top) {
```

Comment:
@LavrovArtem @miherlosev  Guys, we no longer use the `window.self` property, it can be overriden `window.self = {};`, just use `window === window.top` check.